jpo-asn-j2735-2024/update.sh

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env bash
```

Comment:
question: what's this bash script used for? It's called "update" but only checking files out. I also don't see a reference to it in the README

Reply:
That is used during this PR process to pull a subset of the generated classes into this branch from the main branch when the classes are regenerated in the main branch.

Reply:
Deleted it since I think this PR is almost done... but it will back in the next PR..

Reply:
Ah ok, that makes more sense now. When we add it back into the next one can we just have a header describing it? That way we don't question it again